process/block/preprocess/sortedTransactionsProvider.go

```diff
@@ -0,0 +1,65 @@
+package preprocess
+
+import (
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/storage"
+	"github.com/ElrondNetwork/elrond-go/storage/txcache"
+)
+
+// createSortedTransactionsProvider is a "simple factory" for "SortedTransactionsProvider" objects
+func createSortedTransactionsProvider(transactionsPreprocessor *transactions, cache storage.Cacher, cacheKey string) SortedTransactionsProvider {
```

Comment:
transactionsPreprocessor and cacheKey are not used.

Reply:
Fixed, removed leftovers :)